README.md

```diff
@@ -290,64 +290,72 @@ We're using [the built-in django translation module](https://docs.djangoproject.
 1. Ensure that the `i18n` module is loaded at the top of the file:
 
    ```django
-   {% extends 'base.html' %}
+   {% extends 'layouts/base.html' %}
    {% load i18n %}
    ...
    ```
 
-1. Add the translation string to the page:
+2. Add the translation string to the page:
 
-   ```django
+   ```
    <h1>{% translate "namespace.mytranslation" %}</h1>
    ```
 
-1. Update the locale file by running the following command:
+3. Update the locale file by running the following command in a `fab sh` shell:
 
-   ```console
-   python manage.py makemessages -l {langage_code}
    ```
+   python ./manage.py makemessages --no-obsolete --add-location file -l en_GB
+   ```
+
+4. In the generated `.po` file, find the generated translation section, it will be a block like this, with the `msgid` corresponding to the key you added in the template:
 
-   where `language_code` is the ISO 3166-1 country code (e.g. en_GB)
+   ```
+   #: ds_caselaw_editor_ui/templates/includes/my_template.html
+   #, fuzzy
+   #| msgid "namespace.othertranslation"
+   msgid "namespace.mytranslation"
+   msgstr "An existing translation autofilled as an example"
+   ```
 
-1. In the generated `.po` file, find the generated msgid string and add the translation below it:
+   You need to do two things here - first remove the line starting with `#, fuzzy` and any lines starting with `#|` below it, then edit the line starting with `msgstr` to include your translation string. The end result will look something like this:
 
-   ```console
+   ```
+   #: ds_caselaw_editor_ui/templates/includes/my_template.html
    msgid "namespace.mytranslation"
    msgstr "This is my translation"
    ```
 
-1. Compile the translations to a binary file:
-
-   ```console
-   python manage.py compilemessages
+5. Compile the translations to a binary file (this should also be run inside a `fab sh` shell):
+   ```
+   python ./manage.py compilemessages
```

Comment:
This should be either `python manage.py` or `./manage.py`, not both. Probably `python manage.py`.